servant/server/mu-servant-server.cabal

```diff
@@ -0,0 +1,57 @@
+name:               mu-servant-server
+version:            0.4.0.0
+synopsis:           Servant servers for Mu definitions
+description:
+  With @mu-servant-server@ you can easily build Servant servers for mu-haskell!
+
+license:            Apache-2.0
+license-file:       LICENSE
+author:             Temp Name
```

Comment:
You can safely add your self as the author of this package 😉

Reply:
Haha this is the first time I'm authoring a package so I wasn''t sure. I'll update it.